SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,104 +5,241 @@
+            }
+        }
 
-        public IContainerService SecurityServiceTestUIContainer;
+        static string ExecuteBashCommand(string command)
+        {
+            // according to: https://stackoverflow.com/a/15262019/637142
+            // thans to this we will pass everything as one command
+            command = command.Replace("\"", "\"\"");
 
-        public String SecurityServiceTestUIContainerName;
+            var proc = new Process
+                       {
+                           StartInfo = new ProcessStartInfo
+                                       {
+                                           FileName = "/bin/bash",
+                                           Arguments = "-c \"" + command + "\"",
+                                           UseShellExecute = false,
+                                           RedirectStandardOutput = true,
+                                           CreateNoWindow = true
+                                       }
+                       };
+            Console.WriteLine(proc.StartInfo.Arguments);
 
-        public ISecurityServiceClient SecurityServiceClient;
-        
+            proc.Start();
+            proc.WaitForExit();
+
+            return proc.StandardOutput.ReadToEnd();
+        }
+
+        private static IContainerService SetupSecurityServiceContainer(String containerName,
+                                                                      ILogger logger,
+                                                                      String imageName,
+                                                                      INetworkService networkService,
+                                                                      String hostFolder,
+                                                                      Int32 dockerPort,
+                                                                      (String URL, String UserName, String Password)? dockerCredentials,
+                                                                      Boolean forceLatestImage = false,
+                                                                      List<String> additionalEnvironmentVariables = null)
+        {
+            logger.LogInformation("About to Start Security Container");
+
+            List<String> environmentVariables = new List<String>();
+            environmentVariables.Add($"ServiceOptions:PublicOrigin=http://{containerName}:{dockerPort}");
+            environmentVariables.Add($"ServiceOptions:IssuerUrl=http://{containerName}:{dockerPort}");
+            environmentVariables.Add("ASPNETCORE_ENVIRONMENT=IntegrationTest");
+            environmentVariables.Add($"urls=http://*:5551");
+
+            if (additionalEnvironmentVariables != null)
+            {
+                environmentVariables.AddRange(additionalEnvironmentVariables);
+            }
+
+            String containerFolder = FdOs.IsLinux() ? "/home/txnproc/trace" : "C:\\home\\txnproc\\trace";
```

Comment:
Codacy found an issue: [Remove the unused local variable 'containerFolder'.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)